oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/DefaultOAuth2AuthorizedClientManager.java

```diff
@@ -134,13 +133,19 @@ public void setContextAttributesMapper(Function<OAuth2AuthorizeRequest, Map<Stri
 
 		@Override
 		public Map<String, Object> apply(OAuth2AuthorizeRequest authorizeRequest) {
-			Map<String, Object> contextAttributes = Collections.emptyMap();
+			Map<String, Object> contextAttributes = new HashMap<>();
 			String scope = authorizeRequest.getServletRequest().getParameter(OAuth2ParameterNames.SCOPE);
 			if (StringUtils.hasText(scope)) {
-				contextAttributes = new HashMap<>();
 				contextAttributes.put(OAuth2AuthorizationContext.REQUEST_SCOPE_ATTRIBUTE_NAME,
 						StringUtils.delimitedListToStringArray(scope, " "));
 			}
+			String username = authorizeRequest.getServletRequest().getParameter(OAuth2ParameterNames.USERNAME);
```

Comment:
I figured because of this comment: https://github.com/spring-projects/spring-security/pull/7013/files#diff-b1268bb315785679ff599fe121c6a23aR142 this mapping would be removed from both the servlet bits as well as the reactive bits. This PR appears to remove it from reactive but not servlet.

Reply:
I went ahead and logged #7350 to address this separately so this PR can be merged before RC1.

However, I'd recommend that #7350 ticket be addressed before RC1 if possible.